english-handparsed/wsj_9009.mrg

```diff
@@ -0,0 +1,581 @@
+(ROOT
+  (S
+    (S
+      (VP
+        (VBG Returning)
+        (PP
+          (IN to)
+          (NP (NNP Assisi)))))
+    (, ,)
+    (NP (PRP he))
+    (VP
+      (VBD traversed)
+      (NP (DT the) (NN city))
+      (S
+        (VP
+          (VBG begging)
+          (NP
+            (NP (NNS stones))
+            (PP
+              (IN for)
+              (NP
+                (NP (DT the) (NN restoration))
+                (PP
+                  (IN of)
+                  (NP (NNP St.) (NNP Damiano) (POS 's)))))))))
+    (. .)))
+
+
+(ROOT
+  (S
+    (S
+      (VP
+        (VBG Preserving)
```

Comment:
NP of some sort?

Reply:
Actually, quite a few instances of `(S (VP Doing_VBG something)) (VP is ...)` exist in PTB, so maybe this is fine